Quadriga/src/main/java/edu/asu/spring/quadriga/domain/workbench/IPublicPage.java

```diff
@@ -0,0 +1,39 @@
+package edu.asu.spring.quadriga.domain.workbench;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @description   : interface to implement PublicPage class.
+ * 
+ * @author        : Prasanth Priya Nesan
+ *
+ */
+public interface IPublicPage 
+{

```

Comment:
please read the style guide we're using (linked from Confluence) and format accordingly.